R/data.R

```diff
@@ -30,3 +30,71 @@
 #
 ################################################################################
 "hbData"
+
+
+################################################################################
+#
+#'
+#' Urinary iodine (μg/L) values for women and children
+#'
+#' This sample dataset (`iodine_sample`) contained 312 observations and 8 variables.
+#' The detailed variable's description are mentioned in the following table.
+#'
+#' @format A data frame with 8 columns and 312 rows.
+#'
+#'
+#' **Variables** | **Description**
+#' :--- | :---
+#' *psu* | Primary sampling unit identifier
+#' *state* | State identifier
+#' *locality* | Locality identifier
+#' *sex* | Sex (1 = Male; 2 = Female)
+#' *m.age* | Age (in years) of female/mother with ages > 5 years
+#' *ch.age* | Age (in months) of children with ages less than 5 years
+#' *group* | Group classification
+#' *iodine* | Urinary iodine (μg/L) values
+#'
+#' @source will update later!
+#'
+#' @examples
+#' # explore the first 6 observations from the dataset
+#' head(iodine_sample)
+#'
+#
+################################################################################
+"iodine_sample"
+
+################################################################################
+#
+#'
+#' Urinary iodine (μg/L) values for women and children
```

Comment:
update title of this dataset to reflect serum ferritin

Reply:
updated the dataset title and dataset description in the new pull request.